### PR TITLE
[Formatter] Fix test cases and exception handling

### PR DESCRIPTION
**Pull request by IrushiL**
## Purpose
* Fix the  `class_definition_declaration_5.bal` test case.
* Fix indentation in query expressions.
* Handle exceptions while formatting a node in order to proceed formatting the remaining nodes.

## Check List 
- [X] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java [hunk 1]
```diff
@@ -3630,33 +3635,39 @@ public Token transform(Token token) {
...
+            if (shouldWrapLine(oldNode, parent)) {
+                node = wrapLine(oldNode, parent);
+            }
 
-        env.trailingNL = prevTrailingNL;
-        env.trailingWS = prevTrailingWS;
-        return node;
+            env.trailingNL = prevTrailingNL;
+            env.trailingWS = prevTrailingWS;
+            return node;
```
Comment (nadeeshaan) [inline misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java:3665]: Reduce the two return statements (Within try and the catch blocks) to a single one.

File: misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java [hunk 2]
```diff
@@ -3630,33 +3635,39 @@ public Token transform(Token token) {
...
+        } catch (Exception e) {
+            LOGGER.error(String.format("Error while formatting [node: %s] [line: %s]: %s",
```
Comment (nadeeshaan) [inline misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java:3667]: Only the starting line number will not be helpful when there are multiple nodes in a single line, shall we add at least the starting column as well?

**Review by nadeeshaan — CHANGES_REQUESTED** [review]

File: misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java (hunk 1 above)
Comment (IrushiL) [reply misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java]: Fixed

File: misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java (hunk 2 above)
Comment (IrushiL) [reply misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/NewFormattingTreeModifier.java]: Done

**Review by SupunS — APPROVED** [review]

**Pull request merged**